src/components/Heroes/RankColumns.ts

```diff
@@ -0,0 +1,271 @@
+import heroes from 'dotaconstants/build/heroes.json';
+import constants from 'src/components/constants'
+import { abbreviateNumber, displayHeroId } from '../../utility';
+import { GlobalStrings } from 'src/types/common/GlobalStrings'
+import { Heroes } from 'src/types/Hero/Heroes'
```

Comment:
what is the reason to mix relative imports with absolute imports?

Reply:
There was no reason, it was simply copied from the old file